tests/k6/components/utils.js

```diff
@@ -626,10 +626,12 @@ export function applyModelReplicaChange(config) {
         const modelId = Math.floor(Math.random() * config.maxNumModels[j])
         const modelName = config.modelNamePrefix[j] + modelId.toString()
 
-        let replicas =  Math.floor(Math.random() * config.maxModelReplicas[j]) + 1
-        const model = generateModel(config.modelType[j], modelName, 1, replicas, config.isSchedulerProxy, config.modelMemoryBytes[j], config.inferBatchSize[j])
+        const rand = Math.random()
+        const replicas =  Math.floor(rand * config.maxModelReplicas[j]) + 1
+        const memory = (rand > 0.5)? replicas + "k": "1k"   // this will induce a change in memory causing a new version of the model to be created
```

Comment:
more concretely on the change itself: this will set the memory to "replicas" K. I assume we'll always use this with small models, without any actual issue in terms of scheduling onto servers where the memory hint actuall matters.

Reply:
thats a good point, perhaps ideally I should induce a small change to the provided memory by the user.

Reply:
As discussed will merge as it stands and improve in a follow up PR